src/clients/java/src/client.zig

```diff
@@ -571,6 +576,23 @@ const JNIHelper = struct {
         return env;
     }
 
+    pub inline fn try_get_env(vm: *jni.JavaVM) ?*jni.JNIEnv {
+        var env: *jni.JNIEnv = undefined;
+        const jni_result = vm.get_env(&env, jni_version);
+        return switch (jni_result) {
+            .ok => env,
+            .thread_detached => null,
+            else => {
+                const message = "Unexpected result calling JavaVM.GetEnv";
+                log.err(
+                    message ++ "; Error = {} ({s})",
+                    .{ @intFromEnum(jni_result), @tagName(jni_result) },
+                );
+                @panic("JNI: " ++ message);
```

Comment:
Maybe combine into  `std.debug.panic` -- at least right now, clients don't surface any log messages afaik. (Likewise in `create_key`/`set_key`).

Reply:
Yes, moving to `std.debug.panic` is a great idea!

I'll let it "as is" for now just because it's standardized with the rest of the JNI client, and change all in a follow-up PR when the client logging is merged.